package.json

```diff
@@ -10,7 +10,8 @@
     "postcss": "^8.4.47",
     "postcss-cli": "^11.0.0",
     "sass": "^1.80.6",
-    "standard": "^17.1.2"
+    "standard": "^17.1.2",
+    "stimulus-autocomplete": "^3.1.0"
```

Comment:
Why is this going in package.json?

Reply:
Good question. We don't need it there w/ importmap goodness, yeah?